playbooks/shutdown_vms.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Shutdown AAP VMs
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tasks:
+    - name: Shutdown VMs based on VM names
+      azure.azcollection.azure_rm_virtualmachine:
+        resource_group: "{{ resource_group }}"
+        name: "{{ item }}"
```

Comment:
can we filter VMs by tag?

Reply:
hmm.. the azure collection [doc](https://docs.ansible.com/ansible/latest/collections/azure/azcollection/azure_rm_virtualmachine_module.html) states to use name instead of tags. e.g:

```
- name: Power Off
  azure_rm_virtualmachine:
    resource_group: myResourceGroup
    name: testvm002
    started: false
```

Reply:
But it will shutdown all VMs from the resource group

Reply:
And from where this `virtual_machine_name` var will get value?